src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -209,7 +211,7 @@ nav.bd-links {
     }
   }
 
-  li > a {
```

Comment:
it is needed because of the wrapping heading-name and chevron nodes into a flex container, right?  I wouldn't worry about it too much --- we're not yet at v1.0 so we aren't guaranteeing stability (especially CSS/classnames) and if there were downstream overrides, they should be easy to fix (by copying this change) right?